pkgs/applications/misc/nnn/default.nix

```diff
@@ -1,4 +1,9 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, ncurses, readline, conf ? null }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, ncurses, readline
+, conf ? null, withIcons ? false, withNerdIcons ? false }:
+
+# Mutually exclusive options
+assert withIcons -> withNerdIcons == false;
+assert withNerdIcons -> withIcons == false;
 
 with lib;
```

Comment:
It would be a good idea to eventually remove this.

Reply:
The asserts you mean? Why?

Reply:
with lib. You always want to apply with in the smallest scope possible.

Reply:
Ah, makes sense.

I can do it, shouldn't be difficult.

Reply:
Done.